### DISCUSSION TITLE
关于https隧道frpc自动生成TLS证书问题

### DISCUSSION THREAD

**Discussion by izhuolin** [natfrp/wiki#410] [使用问题] [answered]
我选择使用的是https隧道 但是由frpc自动生成的TLS证书会导致网页不安全无法打开，看了文档没有看到安卓自定义证书教学，请问有什么办法可以解决吗

**Comment by fakeboboliu** [comment, marked as answer]
非常感谢您的意见，因为安卓系统限制，您**可能**无法使用启动器实现此功能。
我们将在后续版本中做出相应优化。

对于您目前的使用，我们建议可以使用下面之一解决方案：
 - 在您的被穿透本地应用中配置 HTTPS （推荐，通常这个方法工作的最好，如果它们支持）
 - 以 `/data/data/com.natfrp.launcher/service` 作为工作目录，参考 Linux 教程即可（如果您是熟练 Android 用户，并拥有一台经过 **Root** 的手机）
 - 使用 Termux 或 终端模拟器 运行启动器（如果您是 Linux 用户）
 - 使用海外节点（如果您的需求是绕过我们的强制 HTTPS 要求）
